sros2/setup.py

```diff
@@ -60,13 +60,13 @@ def package_files(directory):
             'sros2.verb = sros2.verb:VerbExtension',
         ],
         'sros2.verb': [
-            'create_key = sros2.verb.create_key:CreateKeyVerb',
```

Comment:
You may want to deprecate `create_key` and `list_keys` rather than having them disappear.

Reply:
You're right, it's a good idea to deprecate for Galactic and remove after. Done in latest push.